### ISSUE TITLE
Building release version for iOS failed

### ISSUE THREAD

**Issue by jinqiupeter** [jaggerwang/zqc-app-demo#6]
When I try to build a release version for iOS, the following error occurred:
```
Undefined symbols for architecture i386:
  "___gxx_personality_v0", referenced from:
      _RCTDefaultLogFunction_block_invoke in libReact.a(RCTLog.o)
      _RCTFormatLog in libReact.a(RCTLog.o)
      ___RCTAddLogFunction_block_invoke in libReact.a(RCTLog.o)
      _RCTPerformBlockWithLogFunction in libReact.a(RCTLog.o)
      _RCTPerformBlockWithLogPrefix in libReact.a(RCTLog.o)
      RCTGetLocalLogFunction() in libReact.a(RCTLog.o)
      ___RCTPerformBlockWithLogPrefix_block_invoke in libReact.a(RCTLog.o)
      ...
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
<img width="891" alt="screen shot 2017-03-08 at 1 26 21 pm" src="https://cloud.githubusercontent.com/assets/12045814/23691408/2fe93738-0403-11e7-8789-94fcb2b536bf.png">
, I tried to upgrade ReactNative to latest version (0.42.0) but didn't help.

I also tried to build release version for vanilla React Native project (generated by react-native init AweSomeProject), it worked properly, so there might be something wrong in the libraries used here, e.g. react-native-wechat, react-native-video etc.

@jaggerwang would you please take a look?

**Comment by jaggerwang** [comment]
Do you use Xcode to package? Which version? And a screenshot will be very helpful.

**Comment by jinqiupeter** [comment]
Yes I used latest Xcode, here is it:
<img width="1440" alt="screen shot 2017-03-09 at 9 45 51 am" src="https://cloud.githubusercontent.com/assets/12045814/23732436/13f5534e-04ae-11e7-8edc-423f77d85725.png">

and running on latest macOS:
<img width="593" alt="screen shot 2017-03-09 at 9 47 27 am" src="https://cloud.githubusercontent.com/assets/12045814/23732492/5a8f080e-04ae-11e7-8461-dc46901fc51d.png">


this is how I switched to Release:
<img width="1440" alt="screen shot 2017-03-09 at 9 46 27 am" src="https://cloud.githubusercontent.com/assets/12045814/23732468/3874fff8-04ae-11e7-8b8c-2c4378f16928.png">

here is the output:
<img width="1440" alt="screen shot 2017-03-09 at 9 49 44 am" src="https://cloud.githubusercontent.com/assets/12045814/23732479/45b5b360-04ae-11e7-8189-e6f2c34d3b94.png">

**Comment by jaggerwang** [comment]
You need choose "Gneric iOS Device" when build.

**Comment by jinqiupeter** [comment]
Still got the same error even with Generic iOS Device:
<img width="1019" alt="screen shot 2017-03-10 at 12 44 51 am" src="https://cloud.githubusercontent.com/assets/12045814/23760515/e63c9840-052a-11e7-8d26-2b5822cfe2e6.png">
,

But then I googled and realized that this is already an [open issue](https://github.com/facebook/react-native/issues/11285) for RN, and there is a workaround for it, so I did what suggested (add -lc++ to Other Linker Options for the test target) and fixed the problem.

Thanks for your time anyways!
Maybe this should be listed in Known Issues?
